src/Microsoft.PowerShell.Commands.Utility/commands/utility/GetUptime.cs

```diff
@@ -0,0 +1,72 @@
+/********************************************************************++
+Copyright (c) Microsoft Corporation.  All rights reserved.
+--********************************************************************/
+
+using System;
+using System.Management.Automation;
+using System.Diagnostics;
+using System.Management.Automation.Internal;
+
+namespace Microsoft.PowerShell.Commands
+{
+    /// <summary>
+    /// This class implements Get-Uptime
+    /// </summary>
+    [Cmdlet(VerbsCommon.Get, "Uptime", DefaultParameterSetName = TimespanParameterSet, HelpUri = "")]
```

Comment:
> $cmdletList = Import-Csv $filePath -ea Stop

@Francisco-Gamino, `$cmdletList` is reading from the test data files, which don't have `Get-Uptime` in them, so it seems `Get-Uptime` will not be hit by this test. Can you please confirm?